### PR TITLE
Update setup-guides-for-microsoft-365.md

### PR DESCRIPTION
**Pull request by msbemba**
Added the new Integrate a third-party cloud app with Azure AD to the documentation

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/4197

File: microsoft-365/enterprise/setup-guides-for-microsoft-365.md [hunk 1]
```diff
@@ -211,3 +211,9 @@ Use the [Deploy Office to remote users guide](https://aka.ms/officeremoteinstall
 ### Deploy and update Microsoft 365 Apps with Configuration Manager
 
 For organizations using Configuration Manager, you can use the [Deploy and update Microsoft 365 Apps with Configuration Manager advisor](https://aka.ms/oppinstall) to generate a script that will automatically configure your Microsoft 365 Apps deployment using best practices recommended by FastTrack engineers. Use this guide to build your deployment groups, customize your Office apps and features, configure dynamic or lean installations, and then run the script to create the applications, automatic deployment rules, and device collections you need to target your deployment. 
+
+### Integrate a third-party cloud app with Azure AD
+
+Improve the user experience and provide an additional layer of security by integrating your third-party app with ‎Azure Active Directory (Azure AD). With this end-to-end experience you can do most of the configuration directly from this wizard. Where appropriate, we'll redirect you to the corresponding configuration page.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/setup-guides-for-microsoft-365.md]:
```suggestion
Improve the user experience and provide an additional layer of security by integrating your third-party app with ‎Azure Active Directory (Azure AD). With this end-to-end experience, you can do most of the configuration directly from this wizard. Where appropriate, we'll redirect you to the corresponding configuration page.
```

File: microsoft-365/enterprise/setup-guides-for-microsoft-365.md [hunk 2]
```diff
@@ -211,3 +211,9 @@ Use the [Deploy Office to remote users guide](https://aka.ms/officeremoteinstall
...
+
+Use the [Integrate third-party cloud app](https://admin.microsoft.com/Adminportal/Home?#/azureadappintegration) wizard to Integrate a third-party cloud app with Azure AD.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/setup-guides-for-microsoft-365.md]:
```suggestion
Use the [Integrate third-party cloud app](https://admin.microsoft.com/Adminportal/Home?#/azureadappintegration) wizard to integrate a third-party cloud app with Azure AD.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**